### PR TITLE
FIX #24945

### LINKED ISSUE #24945: Project event mail dont create after sending mail action
### Bug

No trace of the email sent in the project in the list of events

### Environment Version

16.0.3

### Environment OS

Windows 11

### Environment Web server

Linux

### Environment PHP

7.4.33

### Environment Database

MySQL or MariaDB 5.5.5-10.3.39-MariaDB

### Environment URL(s)

_No response_

### Expected and actual behavior

I would like an event to be added

### Steps to reproduce the behavior

I wrote an email directly in the project and it was not added to the list of events.

### Attached files

_No response_

### PR DESCRIPTION
**Pull request by inoveaconseil**

### CODE REVIEW

File: htdocs/core/triggers/interface_50_modAgenda_ActionsAuto.class.php [hunk 1]
```diff
@@ -835,7 +835,21 @@ public function runTrigger($action, $object, User $user, Translate $langs, Conf
 			$langs->loadLangs(array("agenda", "other", "projects"));
 
 			if (empty($object->actionmsg2)) {
-				$object->actionmsg2 = $langs->transnoentities("ProjectModifiedInDolibarr", $object->ref);
+				$object->actionmsg2 = $langs->transnoentities("ProjectSentByEMail", $object->ref);
```
Comment (frederic34) [inline htdocs/core/triggers/interface_50_modAgenda_ActionsAuto.class.php:838]: action is PROJECT_MODIFY

File: htdocs/core/triggers/interface_50_modAgenda_ActionsAuto.class.php [hunk 2]
```diff
@@ -835,7 +835,21 @@ public function runTrigger($action, $object, User $user, Translate $langs, Conf
...
+
+			$object->sendtoid = 0;
+		} elseif ($action == 'PROJECT_SENTBYMAIL') {
+			// Load translation files required by the page
+			$langs->loadLangs(array("agenda", "other", "projects"));
+
+			if (empty($object->actionmsg2)) {
+				$object->actionmsg2 = $langs->transnoentities("ProjectSentInDolibarr", $object->ref);
 			}
 			$object->actionmsg = $langs->transnoentities("ProjectModifiedInDolibarr", $object->ref);
```
Comment (frederic34) [inline htdocs/core/triggers/interface_50_modAgenda_ActionsAuto.class.php:854]: action is PROJECT_SENTBYMAIL

**Comment by eldy** [comment]
Label of the 2 actions seems to have been inverted...

**Comment by aspangaro** [comment]
Already fix in v17. Need to modify database. Not reportable in v16.

https://github.com/Dolibarr/dolibarr/commit/ea8759e627cef4cc15b50c9a2d7d9128e5c5f911

**Pull request closed without merging**